pkg/rulefmt/testdata/test.yaml

```diff
@@ -0,0 +1,59 @@
+version: 1
+groups:
+- name: my-group-name
+  interval: 30s   # defaults to global interval
+  rules:
+  - alert: HighErrors
+    expr: |
+      sum without(instance) (rate(errors_total[5m]))
+      / sum without(instance) (rate(requests_total[5m]))
```

Comment:
Ugh, is this confirmed? Any workaround? This would be a serious drawback in readability.